### PR TITLE
Week 4

### PR DESCRIPTION
**Pull request by dlfivefifty**

### CODE REVIEW

**Comment by dlfivefifty** [comment]
@SamBrzezicki These are ready for review please

File: notes/II.4.Intervals.tex [hunk 1]
```diff
@@ -0,0 +1,85 @@
+
+\section{Interval Arithmetic}
+It is possible to choose the rounding mode (up/down/nearest/etc.) which we can use to do rigorous computation to compute bounds on, for example, computing the digits of $\E$. To do this we will use set/interval arithmetic. For sets $X,Y \ensuremath{\subseteq} \ensuremath{\bbR}$, the set arithmetic operations are defined as
+\[
+X + Y := \{x + y : x \ensuremath{\in} X, y \ensuremath{\in} Y\}, XY := \{xy : x \ensuremath{\in} X, y \ensuremath{\in} Y\}, X/Y := \{x/y : x \ensuremath{\in} X, y \ensuremath{\in} Y\}
```
Comment (SamBrzezicki) [inline notes/II.4.Intervals.tex]: Maybe put these on separate lines for clarity

File: notes/II.4.Intervals.tex [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+\]
+We will use floating point arithmetic to construct approximate set operations \ensuremath{\oplus}, \ensuremath{\otimes} so that
+\[
+  X + Y \ensuremath{\subseteq} X \ensuremath{\oplus} Y, XY \ensuremath{\subseteq} X \ensuremath{\otimes} Y, X/Y \ensuremath{\subseteq} X \ensuremath{\oslash} Y
```
Comment (SamBrzezicki) [inline notes/II.4.Intervals.tex]: These on separate lines too?

File: sheets/sheet4.tex [hunk 3]
```diff
@@ -0,0 +1,185 @@
...
+\[
+{f^{\rm FP}(x+h) \ensuremath{\ominus} f^{\rm FP}(x-h) \over  2h} = f'(x) + \ensuremath{\varepsilon}
+\]
+show the absolute error is bounded by
+\[
+\ensuremath{\varepsilon} \ensuremath{\leq} {|f'(x)| \over 2} \ensuremath{\epsilon}_{\rm m} + {M \over 3} h^2 + {2 c \ensuremath{\epsilon}_{\rm m} \over h},
+\]
+where we assume $x \ensuremath{\in} [0,1]$, $h = 2^{-n}$ for $n \ensuremath{\leq} S$ so that $x \ensuremath{\oplus} h  = x+ h$ and $x \ensuremath{\ominus} h = x-h$.
+
+\textbf{Problem 5(a)} For intervals  $X = [a,b]$ and $Y = [c,d]$ satisfying $0 < a < b$ and $0 < c < d$, and $n > 0$ prove that
```
Comment (SamBrzezicki) [inline sheets/sheet4.tex]: No problem 5(b) so either rename to 5 or add 5(b) on the second part later

**Pull request merged**